packages/cli/tests/lib/output.js

```diff
@@ -16,6 +17,9 @@ function tmpDir() {
 async function subject(name) {
 	let src = resolve(subjects, name);
 	let dest = tmpDir();
+	if (!existsSync(dest)) {
+		mkdirSync(dest, { recursive: true });
+	}
```

Comment:
Running individual test suites (i.e., `yarn test tests/build.test.js`) results in the output dir not being created and the tests exiting when trying to copy the subjects over. This corrects that.

Reply:
wondering if it would make sense to move this into `tmpDir`

Reply:
I do think that makes more sense